monitor-e2e/utils.js

```diff
@@ -0,0 +1,42 @@
+const shell = require('shelljs')
+const Web3 = require('web3')
+const { ERC677_BRIDGE_TOKEN_ABI } = require('../commons')
+
+const checkAll = () => {
+  // the different prefixes come from macOS/linux docker differences and/or different docker-compose versions
+  ;['e2e-commons_', 'e2e_commons_', 'e2ecommons_'].forEach(prefix =>
+    ['monitor_1', 'monitor-erc20_1', 'monitor-erc20-native_1'].forEach(container =>
+      shell.exec(`docker exec ${prefix}${container} yarn check-all`)
```

Comment:
I need to run `docker exec e2e-commons_monitor_1` on macOS, and `docker exec e2ecommons_monitor_1` on CI.
I'm not exactly sure if the difference comes from macOS/linux difference, or docker-compose/docker version difference.
The `forEach` is supposed to try all combinations.